server/gorillamux/problems/methodnotfound.go

```diff
@@ -0,0 +1,32 @@
+package problems
```

Comment:
I started to include the .md files with the problems, so the consuming APIs can just copy and paste the template into their Problem docs.

Reply:
kk